### PR TITLE
TrackBar, Slider, RangeSlider, SliderHandle, Tooltip: move files to TS

### PR DESCRIPTION
**Pull request by EugeniyKiyashko**

### CODE REVIEW

File: packages/devextreme/js/__internal/scheduler/appointments/m_appointment.ts [hunk 1]
```diff
@@ -11,8 +11,8 @@ import { addNamespace } from '@js/events/utils/index';
 import dateLocalization from '@js/localization/date';
 import messageLocalization from '@js/localization/message';
 import Resizable from '@js/ui/resizable';
-import { hide, show } from '@js/ui/tooltip/ui.tooltip';
 
+import { hide, show } from '../../ui/tooltip/m_tooltip';
```
Comment (pomahtri) [inline packages/devextreme/js/__internal/scheduler/appointments/m_appointment.ts]: change please to absolute path, e.g. `'@ts/ui/tooltip/m_tooltip'`

File: packages/devextreme/js/__internal/ui/m_tooltip.ts [hunk 2]
```diff
@@ -1,96 +1,59 @@
-import $ from '../../core/renderer';
-import Guid from '../../core/guid';
-import registerComponent from '../../core/component_registrator';
-import { extend } from '../../core/utils/extend';
-import Popover from '../popover/ui.popover';
+import registerComponent from '@js/core/component_registrator';
```
Comment (pomahtri) [inline packages/devextreme/js/__internal/ui/m_tooltip.ts:1]: can we move this file to tooltip folder?

Comment (EugeniyKiyashko) [reply packages/devextreme/js/__internal/ui/m_tooltip.ts:1]: I will do it in the next PR to ensure that the history is saved correctly.

File: packages/devextreme/js/__internal/scheduler/appointments/m_appointment.ts (hunk 1 above)
Comment (EugeniyKiyashko) [reply packages/devextreme/js/__internal/scheduler/appointments/m_appointment.ts]: resolved

**Review by Zedwag — APPROVED** [review]

**Review by pomahtri — APPROVED** [review]

**Pull request merged**